Makefile

```
@@ -15,7 +15,7 @@ PYTHON_EXECUTABLE_PREFIX := $(shell test -d "$${VIRTUALENV_ROOT}" && echo "$${VI
 
 .PHONY: bootstrap
 bootstrap: generate-version-file ## Set up everything to run the app
-	pip3 install -r requirements_for_test.txt
+	uv pip sync requirements_for_test.txt
 	createdb notification_api || true
 	(. environment.sh && flask db upgrade) || true
```

in my new `uv` managed world i dont have an accessible venv - rather uv wants you to use `uv run` to access things from within the venv. So if i run `make bootstrap` locally i now get `/bin/bash: flask: command not found. i can't even run `which python` or `which pip` on my machine - a python dev with no available python! 🤯 

The change would be:

```suggestion
	(. environment.sh && uv run --no-project flask db upgrade) || true
```

(we'd also want to update all the other bits like when we run python `python -c "...  copy_config()"` in freeze-reqs)

not sure if we need to do that now? i can work around it/deal with it. but just something to think about. somewhat surprised it works in docker 🤔

---

you know what, i can work around this by just running `source .venv/bin/activate` - i'll stick that in my direnv and leave it at that